# -*- coding: utf-8 -*-

from PySide2.QtCore import *
from PySide2.QtGui import *
from PySide2.QtWidgets import *

class Ui_Form(object):
    def setupUi(self, Form):
        if not Form.objectName():
            Form.setObjectName("Form")
        Form.resize(320, 200)
        icon = QIcon()
        icon.addFile("small_black.png", QSize(), QIcon.Normal, QIcon.Off)
        Form.setWindowIcon(icon)
        Form.setStyleSheet("QGroupBox {border: 1px solid rgb(0, 170, 0); border-radius: 5px;}"
            "QLabel {font: 12pt \"Arial\"; color: rgb(0, 0, 127);}"
            "QPushButton {padding: .7em 1.2em; color: rgb(85, 0, 127); font: 75 bold 12pt \"Tahoma\";}")

        self.centralwidget = QWidget(Form)
        self.centralwidget.setObjectName(u"centralwidget")
        self.verticalLayout_3 = QVBoxLayout(self.centralwidget)
        self.verticalLayout_3.setObjectName(u"verticalLayout_3")
        self.groupBox_cursor = QGroupBox(self.centralwidget)
        self.groupBox_cursor.setObjectName(u"groupBox_cursor")
        self.horizontalLayout_2 = QHBoxLayout(self.groupBox_cursor)
        self.horizontalLayout_2.setObjectName(u"horizontalLayout_2")
        self.horizontalLayout_2.setSizeConstraint(QLayout.SetDefaultConstraint)
        self.GET_POS = QPushButton(self.groupBox_cursor)
        self.GET_POS.setObjectName(u"GET_POS")
        self.GET_POS.setMinimumSize(QSize(0, 0))
        self.GET_POS.setSizeIncrement(QSize(0, 0))
        font = QFont()
        font.setFamily(u"Tahoma")
        font.setPointSize(12)
        font.setBold(True)
        font.setItalic(False)
        font.setWeight(75)
        self.GET_POS.setFont(font)
        self.GET_POS.setCursor(QCursor(Qt.PointingHandCursor))
        self.GET_POS.setIconSize(QSize(24, 24))
        self.GET_POS.setFlat(False)
        self.horizontalLayout_2.addWidget(self.GET_POS)
        self.horizontalLayout_pos = QHBoxLayout()
        self.horizontalLayout_pos.setObjectName(u"horizontalLayout_pos")
        self.verticalLayout_x = QVBoxLayout()
        self.verticalLayout_x.setObjectName(u"verticalLayout_x")
        self.label_x = QLabel(self.groupBox_cursor)
        self.label_x.setObjectName(u"label_x")
        self.label_x.setAlignment(Qt.AlignCenter)
        self.verticalLayout_x.addWidget(self.label_x)
        self.line_x = QFrame(self.groupBox_cursor)
        self.line_x.setObjectName(u"line_x")
        self.line_x.setFrameShape(QFrame.HLine)
        self.line_x.setFrameShadow(QFrame.Sunken)
        self.verticalLayout_x.addWidget(self.line_x)
        self.pos_X = QLabel(self.groupBox_cursor)
        self.pos_X.setObjectName(u"pos_X")
        font1 = QFont()
        font1.setFamily(u"Arial")
        font1.setPointSize(12)
        font1.setBold(False)
        font1.setItalic(False)
        font1.setWeight(50)
        self.pos_X.setFont(font1)
        self.pos_X.setAlignment(Qt.AlignCenter)
        self.verticalLayout_x.addWidget(self.pos_X)
        self.horizontalLayout_pos.addLayout(self.verticalLayout_x)
        self.verticalLayout_y = QVBoxLayout()
        self.verticalLayout_y.setObjectName(u"verticalLayout_y")
        self.label_y = QLabel(self.groupBox_cursor)
        self.label_y.setObjectName(u"label_y")
        self.label_y.setAlignment(Qt.AlignCenter)
        self.verticalLayout_y.addWidget(self.label_y)
        self.line_y = QFrame(self.groupBox_cursor)
        self.line_y.setObjectName(u"line_y")
        self.line_y.setFrameShape(QFrame.HLine)
        self.line_y.setFrameShadow(QFrame.Sunken)
        self.verticalLayout_y.addWidget(self.line_y)
        self.pos_Y = QLabel(self.groupBox_cursor)
        self.pos_Y.setObjectName(u"pos_Y")
        self.pos_Y.setFont(font1)
        self.pos_Y.setAlignment(Qt.AlignCenter)
        self.verticalLayout_y.addWidget(self.pos_Y)
        self.horizontalLayout_pos.addLayout(self.verticalLayout_y)
        self.horizontalLayout_2.addLayout(self.horizontalLayout_pos)
        self.verticalLayout_3.addWidget(self.groupBox_cursor)
        self.horizontalLayout = QHBoxLayout()
        self.horizontalLayout.setObjectName(u"horizontalLayout")
        self.groupBox_key = QGroupBox(self.centralwidget)
        self.groupBox_key.setObjectName(u"groupBox_key")
        self.groupBox_key.setEnabled(True)
        sizePolicy = QSizePolicy(QSizePolicy.Preferred, QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.groupBox_key.sizePolicy().hasHeightForWidth())
        self.groupBox_key.setSizePolicy(sizePolicy)
        self.groupBox_key.setFlat(False)
        self.horizontalLayout_6 = QHBoxLayout(self.groupBox_key)
        self.horizontalLayout_6.setObjectName(u"horizontalLayout_6")
        self.key_name = QLabel(self.groupBox_key)
        self.key_name.setObjectName(u"key_name")
        sizePolicy1 = QSizePolicy(QSizePolicy.Fixed, QSizePolicy.Preferred)
        sizePolicy1.setHorizontalStretch(0)
        sizePolicy1.setVerticalStretch(0)
        sizePolicy1.setHeightForWidth(self.key_name.sizePolicy().hasHeightForWidth())
        self.key_name.setSizePolicy(sizePolicy1)
        self.key_name.setMinimumSize(QSize(80, 0))
        self.key_name.setSizeIncrement(QSize(0, 0))
        self.key_name.setBaseSize(QSize(0, 0))
        self.key_name.setFont(font1)
        self.key_name.setAlignment(Qt.AlignCenter)
        self.horizontalLayout_6.addWidget(self.key_name)
        self.horizontalLayout.addWidget(self.groupBox_key)
        self.groupBox_click = QGroupBox(self.centralwidget)
        self.groupBox_click.setObjectName(u"groupBox_click")
        sizePolicy2 = QSizePolicy(QSizePolicy.Expanding, QSizePolicy.Preferred)
        sizePolicy2.setHorizontalStretch(0)
        sizePolicy2.setVerticalStretch(0)
        sizePolicy2.setHeightForWidth(self.groupBox_click.sizePolicy().hasHeightForWidth())
        self.groupBox_click.setSizePolicy(sizePolicy2)
        self.horizontalLayout_3 = QHBoxLayout(self.groupBox_click)
        self.horizontalLayout_3.setObjectName(u"horizontalLayout_3")
        self.GET_CLICK = QPushButton(self.groupBox_click)
        self.GET_CLICK.setObjectName(u"GET_CLICK")
        self.GET_CLICK.setFont(font)
        self.GET_CLICK.setCursor(QCursor(Qt.PointingHandCursor))
        icon1 = QIcon()
        icon1.addFile(u"small_green.png", QSize(), QIcon.Normal, QIcon.Off)
        self.GET_CLICK.setIcon(icon1)
        self.GET_CLICK.setIconSize(QSize(24, 24))
        self.horizontalLayout_3.addWidget(self.GET_CLICK)
        self.click_sum = QLabel(self.groupBox_click)
        self.click_sum.setObjectName(u"click_sum")
        self.click_sum.setFont(font1)
        self.click_sum.setAlignment(Qt.AlignCenter)
        self.horizontalLayout_3.addWidget(self.click_sum)
        self.horizontalLayout.addWidget(self.groupBox_click)
        self.verticalLayout_3.addLayout(self.horizontalLayout)
        Form.setCentralWidget(self.centralwidget)

        self.retranslateUi(Form)

        self.GET_POS.clicked.connect(self.pos_X.update)
        self.GET_POS.clicked.connect(self.pos_Y.update)
        self.GET_CLICK.clicked.connect(self.click_sum.update)

        QMetaObject.connectSlotsByName(Form)
    # setupUi

    def retranslateUi(self, Form):
        Form.setWindowTitle(QCoreApplication.translate("Form", u"Clicker", None))
        self.groupBox_cursor.setTitle(QCoreApplication.translate("Form", u"Cursor", None))
        self.GET_POS.setText(QCoreApplication.translate("Form", u"POSITION", None))
        self.label_x.setText(QCoreApplication.translate("Form", u"X", None))
        self.pos_X.setText(QCoreApplication.translate("Form", u"x", None))
        self.label_y.setText(QCoreApplication.translate("Form", u"Y", None))
        self.pos_Y.setText(QCoreApplication.translate("Form", u"y", None))
        self.groupBox_key.setTitle(QCoreApplication.translate("Form", u"Key name", None))
        self.key_name.setText(QCoreApplication.translate("Form", u"...", None))
        self.groupBox_click.setTitle(QCoreApplication.translate("Form", u"Test clicker", None))
        self.GET_CLICK.setText(QCoreApplication.translate("Form", u" Click", None))
        self.click_sum.setText(QCoreApplication.translate("Form", u"0", None))
    # retranslateUi
